institutions/serializers.py

```diff
@@ -0,0 +1,18 @@
+"""Serializers for the institutions app."""
+
+from rest_framework import serializers
+
+from institutions.models import Institution
+
+
+class InstitutionSerializer(serializers.ModelSerializer):
+    """Serializer for the Institution model."""
+    domains = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Institution
+        exclude = ('id', )
```

Comment:
I gave this one some thought, and I decided that since an institution is highly unique and unique by name, an ID is a useless field to know. We can always identify an institution by its name, and would be better served by searching for it by name on the admin page anyway. And ID field for this model would just be clutter.

Reply:
Even in a wacky case where maybe there are two University of Waterloo in the world, we would need a way to distinguish them anyway, maybe by address or a new location field down the line, an id isn't worth it I think for the admin page.